user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_external_authentication_via_an_identity_provider_using_SAML.md

```diff
@@ -101,6 +101,36 @@ To create a DataMiner metadata file (also referred to as *Service Provider Metad
 > The ``WantAssertionsSigned`` flag is supported as from DataMiner version 10.2.1/10.2.0. If you are using an older version, then set this to false.
 > SAML responses without signatures can be freely edited to tamper with permissions on the application, leading to severe vulnerabilities. We **highly recommend** setting ``WantAssertionsSigned`` to *true* to mitigate this.
 
+## Additional configuration for systems connected to dataminer.services
+
+When your DataMiner System is connected to dataminer.services, additional configuration is required for both the *spMetadata.xml* file and the identity provider.
+
+1. Add the following URLs:
+
+   - `https://<dms-dns-name>-<organization-name>/API/`
+   - `https://<dms-dns-name>-<organization-name>/account-linking`
+   - `https://<dms-dns-name>-<organization-name>/account-linking/`
+
+   Example of remote access URL: <https://dataminer-skyline.on.dataminer.services>
+
+   In this example, the DMS DNS name is "dataminer" and the organization name is "skyline".
+
```

Comment:
I'm not understanding how this remote access URL is an example of one of the three URLs you have listed above. It doesn't end with "/API/", "/account-linking", or "/account-linking/". Also, as you have replaced `<organization-name>` with "skyline.on.dataminer.services", is this then not the organization name instead of just "skyline"?

Reply:
It's an example of a remote access url which is the base url, both `/API` and `/account-linking` are different paths.

In the listing of the urls at line 110 I forgot to add `.on.dataminer.services`, will update it

Reply:
Pushed commit that adds the `.on.dataminer.services` to the URLs